torch/_tensor_docs.py

```diff
@@ -1068,6 +1068,7 @@ def scale_channels(input, scale):
 See :func:`torch.floor`
 """)
 
+
```

Comment:
Forgot the tensor version document?

Reply:
Oh unintended change here, thanks. It's function-only per @gchanan 's suggestion.